python/tk_multi_loader/delegate_publish.py

```diff
@@ -143,12 +143,6 @@ def _on_before_selection(self, widget, model_index, style_options):
         elif is_folder:
             # a folder widget with shotgun data
             widget.set_actions(self._action_manager.get_actions_for_folder(sg_item))
-        else:
```

Comment:
This was already broken half of the time before this branch, so taking it out as Qt doesn't allow to have widget editor tooltips when the standard model item has a tooltip of its own.